plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -513,6 +462,10 @@ fu_plugin_uefi_capsule_coldplug_device (FuPlugin *plugin, FuUefiDevice *dev, GEr
 		fu_device_add_private_flag (FU_DEVICE (dev),
 					    FU_UEFI_DEVICE_FLAG_USE_LEGACY_BOOTMGR_DESC);
 	}
+	if (fu_plugin_has_custom_flag (plugin, "boot-order-lock")) {
```

Comment:
should this be `supports-boot-order-lock`?

Reply:
Nah, the existing plugin one doesn't have this -- we could certainly change the quirk file to match if you think it would be clearer.

Reply:
I've made them both `supports-boot-order-lock` as it's confusing to have both.